app/models/wakes/location.rb

```diff
@@ -13,7 +14,8 @@ def label
     path
   end
 
-  def url(protocol: 'http', host: ENV['DEFAULT_HOST'])
+  def url(protocol: 'http', host: nil)
+    host ||= self.host || ENV['DEFAULT_HOST']
```

Comment:
@gaganawhad @benhutton If we prefer, there is some precedent for overriding the default host altogether. This might be a good practice in general when testing the social metrics (but not redirect) parts of Wakes.

https://github.com/desiringgod/wakes/blob/develop/spec/models/wakes/location_spec.rb#L36-L43